requirements-apps-api.txt

```diff
@@ -1,7 +1,8 @@
 flask==2.2.5
 Flask-Cors==4.0.0
-jsonschema==4.17.3
-openapi-core==0.14.5
+jsonschema==4.20.0
+# TODO convert this pin back to the normal format after the next PyPI release
+openapi-core @ git+https://github.com/python-openapi/openapi-core@0.19.0a1
```

Comment:
Do we have an issue for this, or will dependabot pick it up?

Reply:
Yeah I created an issue for it because I'm not sure what dependabot will do: https://github.com/ASFHyP3/hyp3/issues/2003